src/runtime/gecko/L20n.jsm

```diff
@@ -0,0 +1,5 @@
+import { ResourceBundle } from './resourcebundle';
+
+this.EXPORTED_SYMBOLS = ["ResourceBundle"];
+
+this.ResourceBundle = ResourceBundle;
```

Comment:
I don't think there's a good enough reason to put resource bundles in their own file.  Could simply `import` them in each of the entry points so that rollup bundles them in the dist file?